src/meson.build

```diff
@@ -245,28 +245,51 @@ fwupdtool = executable(
   install_dir: bindir
 )
 
-if get_option('man')
+if pandoc.found()
+  conf_man = configuration_data()
+  conf_man.set('PACKAGE_VERSION', fwupd_version)
   if build_daemon
-    configure_file(
-      input: 'fwupdmgr.1',
+    fwupdmgr_md = configure_file(
+      input: 'fwupdmgr.md',
+      output: 'fwupdmgr.md',
+      configuration: conf_man,
+    )
+    custom_target('fwupdmgr.1',
+      input: fwupdmgr_md,
       output: 'fwupdmgr.1',
-      configuration: conf,
+      command: [
+        pandoc, '--standalone', '--to', 'man', '@INPUT@', '-o', '@OUTPUT@',
+      ],
       install: true,
       install_dir: join_paths(mandir, 'man1'),
     )
-    configure_file(
-      input: 'fwupdagent.1',
+    fwupdagent_md = configure_file(
+      input: 'fwupdagent.md',
+      output: 'fwupdagent.md',
+      configuration: conf_man,
+    )
+    custom_target('fwupdagent.1',
```

Comment:
Shouldn't fwupdagent be hidden behind `compat_cli`?

Reply:
yup, fixed, good catch.